src/data/finset.lean

```diff
@@ -509,6 +518,9 @@ by simpa only [subset_iff, mem_sdiff, and_imp] using λ a m₁ m₂, and.intro (
 @[simp] lemma coe_sdiff (s₁ s₂ : finset α) : ↑(s₁ \ s₂) = (↑s₁ \ ↑s₂ : set α) :=
 set.ext $ λ _, mem_sdiff
 
+lemma to_set_sdiff (s t : finset α) : (s \ t).to_set = s.to_set \ t.to_set :=
```

Comment:
Is this a simp-lemma?

Reply:
About finset-simp lemmas: I don't see a simp lemma rewriting `s.to_set` to `(↑s : set _)` or the other way around, so I'm not sure what the "canonical form" of this function is.

Reply:
@digama0 Is there a general mathlib preference here?

Reply:
I've never heard of `s.to_set` before this (and I wrote it). It's definitely written `↑s` everywhere.